fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkEtlJobHandler.java

```diff
@@ -70,15 +72,16 @@
 public class SparkEtlJobHandler {
     private static final Logger LOG = LogManager.getLogger(SparkEtlJobHandler.class);
 
-    private static final String APP_RESOURCE_NAME = "palo-fe.jar";
     private static final String CONFIG_FILE_NAME = "jobconfig.json";
-    private static final String APP_RESOURCE_LOCAL_PATH = PaloFe.DORIS_HOME_DIR + "/lib/" + APP_RESOURCE_NAME;
     private static final String JOB_CONFIG_DIR = "configs";
     private static final String ETL_JOB_NAME = "doris__%s";
     // 5min
     private static final int GET_APPID_MAX_RETRY_TIMES = 300;
     private static final int GET_APPID_SLEEP_MS = 1000;
 
+    // cluster_id -> lock
+    private static final ConcurrentMap<Integer, Object> DPP_LOCK_MAP = Maps.newConcurrentMap();
```

Comment:
Use SparkResource' name as Key, not ClusterId.
ClusterId is same all the way.

And I think this lock map is unnecessary. You can just call `resource.prepareDppArchive()`.

And `prepareDppArchive()` is protected by `synchronized`.